integration-cli/docker_cli_rmi_test.go

```diff
@@ -308,8 +308,13 @@ RUN echo 2 #layer2
 }
 
 func (*DockerSuite) TestRmiParentImageFail(c *check.C) {
-	parent := inspectField(c, "busybox", "Parent")
-	out, _, err := dockerCmdWithError("rmi", parent)
+	_, err := buildImage("test", `
```

Comment:
I had to update this test because when we use `docker pull` to get the image, the "Parent" field is not set.
It also seems silly to assume that this field even exists much less that busybox has a parent.